scripts/uberenv/packages/geosx/package.py

```diff
@@ -119,6 +119,8 @@ class Geosx(CMakePackage, CudaPackage):
 
     depends_on('pugixml@1.8: +shared')
 
+    depends_on('fmt@8.0: +cxxstd=14 +pic')
```

Comment:
Nice, thank you!

Reply:
@corbett5 This will probably actually break the spack build, because the target branch in your fork does not have `8.x` version of `fmt`. How much effort would it be to rebase your branch onto latest upstream spack/develop and can I help?

Reply:
A non-trivial amount. I tried it before and encountered a bug with Silo, and maybe some other things I don't remember. My path (hack) forward has been to just update the individual package files. This isn't sustainable, but no-one besides me really uses Spack anyways :(

https://github.com/spack/spack/issues/22464